src/Manifest/ManifestManager/Options/WriteTableManifestOptions.php

```diff
@@ -0,0 +1,127 @@
+<?php declare(strict_types = 1);
+
+namespace Keboola\Component\Manifest\ManifestManager\Options;
+
+class WriteTableManifestOptions
+{
+    /** @var string */
+    private $destination;
+
+    /** @var string[] */
+    private $primaryKeyColumns;
+
+    /** @var string[] */
+    private $columns;
+
+    /** @var bool */
+    private $incremental;
+
+    /** @var mixed[][] */
+    private $metadata;
+
+    /** @var mixed[][] */
+    private $columnMetadata;
+
+    /** @var string */
+    private $delimiter;
+
+    /** @var string */
+    private $enclosure;
+
+    /**
+     * @return mixed[]
+     */
+    public function toArray(): array
+    {
+        $result = [];
+        if (isset($this->destination)) {
```

Comment:
Tu by som asi pouzil `foreach (["destination" => "destination", ... ] as $from => $to) { if (isset(...`. No nejako extra mi to nevadi.

Reply:
Taková (jakoby složitá) logika mate tooly. Takže to pak hází třeba "unused private variable" atp,

Reply:
Tak kod piseme pre nas, nie pre tooly, nie? :stuck_out_tongue:

Reply:
Já právě píšu spíš pro tooly, aby to co nejlíp pochopili a tím pádem abych ty věci nemusel hlídat já 🐌  Takhle když přidám novou private proměnnou, tak se mi vysvítí, že jsem ji zapomněl přidat do `toArray()`...